### DISCUSSION TITLE
PTH118: why not `joinpath` instead of `/`?

### DISCUSSION THREAD

**Discussion by leon1995** [astral-sh/ruff#7036] [Ideas]
why ruff recommends "`os.path.join()` should be replaced by `Path` with `/` operator" and not `Path().joinpath`? 

> `/` is more readable than `os.path.join`

[says ruff's documentation](https://beta.ruff.rs/docs/rules/os-path-join/) but `joinpath` is kinda more readable than `/`. So why not recommending `joinpath` instead?

**Comment by leon1995** [comment]
ruffs documentation actually recommends using `joinpath` in the text but not in the example and the error `PTH118` when running `ruff check`.

**Comment by tjkuson** [comment]
I prefer the forward slash notation, as that is how I am used to reading paths on my OS. I also think there is less visual clutter and more consistency in `p1 / p2 / p3` than `p1.joinpath(p2, p3)`. My understanding is that the ruff documentation mentions `Path.joinpath()` as it is the descriptive name for the special `/` path operator.

**Reply by leon1995** [reply]
ok, maybe I got confused then. I think its also a personal choice. Maybe a second example could be added using the actual `joinpath`?
